edx-platform/pearson-vue-theme/lms/templates/index.html

```diff
@@ -11,9 +11,17 @@
     <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
   <div class="signin-container">
     <div class="toggle-form">
-      <h1>${_("Welcome!")}</h1>
-      <br>
-      <span class="text">${_("Sign in or create an account to access your courseware.")}</span>
+      <span class="text">${_("Welcome to CertPREP courseware!")}</span>
+      <span class="text-bold">${_("Everything in one spot for a seamless experience.")}</span>
+      <span class="text">${_("All text, videos, quizzes, practice tests and labs are integrated within the courseware with one consitent interface and log-in for instructors and students.")}</span>
```

Comment:
Shouldn't it be **consistent** instead of consitent?. @alexjmpb

Reply:
yes! thanks for spotting that